app/controllers/concerns/sorting.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Sorting
+  def sort_collection(items, default_sort, default_direction = :asc)
+    field, direction = sort_params(default_sort, default_direction)
+    sort_with_public_send items, field, direction
+  end
+
+  def sort_params(default_sort_params, default_direction = :asc)
+    if params['sort']
+      params['sort'].split.map { |s| s.downcase.to_sym }
+    else
+      [default_sort_params, default_direction]
```

Comment:
### Variable name consistency

`default_sort_params` should probably just be `default_sort`. There's no need for the `_params` suffix.

```suggestion
  def sort_params(default_sort, default_direction = :asc)
    if params['sort']
      params['sort'].split.map { |s| s.downcase.to_sym }
    else
      [default_sort, default_direction]
```